Assets/Features/Managers/GameManager.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
+
+namespace Features.Managers
+{
+    public class GameManager : MonoBehaviour
+    {
+        public static GameManager Instance;
+        
+        public AudioManager.Scripts.AudioManager AudioManager;
```

Comment:
Weird reference to the AudioManager `AudioManager.Scripts.AudioManager`. Might be nicer to just say `AudioManager` instead

Reply:
Added a better reference in using. This is because of the already existing autio manager of unity